src/contracts/core/RewardsCoordinator.sol

```diff
@@ -280,12 +309,28 @@ contract RewardsCoordinator is
         uint16 split
     ) external onlyWhenNotPaused(PAUSED_OPERATOR_PI_SPLIT) checkCanCall(operator) {
         uint32 activatedAt = uint32(block.timestamp) + activationDelay;
-        uint16 oldSplit = _getOperatorSplit(operatorPISplitBips[operator]);
-        _setOperatorSplit(operatorPISplitBips[operator], split, activatedAt);
+        uint16 oldSplit = _getOperatorSplit(_operatorPISplitBips[operator]);
+        _setOperatorSplit(_operatorPISplitBips[operator], split, activatedAt);
 
         emit OperatorPISplitBipsSet(msg.sender, operator, activatedAt, oldSplit, split);
     }
 
+    /// @inheritdoc IRewardsCoordinator
+    function setOperatorDirectedOperatorSetSplit(
+        address operator,
+        OperatorSet calldata operatorSet,
+        uint16 split
+    ) external onlyWhenNotPaused(PAUSED_OPERATOR_SET_OPERATOR_SPLIT) checkCanCall(operator) {
+        require(allocationManager.isOperatorSet(operatorSet), InvalidOperatorSet());
+        require(split <= ONE_HUNDRED_IN_BIPS, SplitExceedsMax());
```

Comment:
You don't need this check here anymore. We moved the 100% check to `_setOperatorSplit` as part of the hotfix for Rewards v2. It should be available in your branch too. Just double confirm.